spec/features/search_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+RSpec.feature 'searching for a place' do
+  describe "a user searches with ambiguous text", :js => true do
+    it "returns multiple matches" do
+      FollowManager.stub(:is_following?).and_return(false)
+      Comment.stub(:for_a_place).and_return(nil)
+
+      VCR.use_cassette("multiple results") do
+        visit "/"
+
+        page.fill_in 'nav-search',
+          :with => 'fresh '
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.